contracts/domain/BosonTypes.sol

```diff
@@ -64,7 +64,7 @@ contract BosonTypes {
         CommitToOffer,
         Exchange,
         Funds,
-        RaiseDispute,
```

Comment:
why no raise dispute?

Reply:
RaiseDispute is obsolete, not used anymore. It was not removed because it would have create a "hole" in the enum values.
Now we have a new value in the enum so, instead of adding a new label/value, we replace this old one with the new one

Reply:
see Klemen comment here: https://github.com/bosonprotocol/boson-protocol-contracts/pull/788#discussion_r1311279105